scipy/stats/tests/test_stats.py

```diff
@@ -7663,6 +7663,20 @@ def test_error_code(self):
             v_values = rng.random(size=(2, 2))
             _ = stats.wasserstein_distance(u_values, v_values)
 
+    def test_flatten(self):
```

Comment:
Consider
```suggestion
    def test_optimization_vs_analytical(self):
```
Or something else that is more descriptive